test/integration/dex-aggregators-integration.spec.ts

```diff
@@ -2,27 +2,28 @@ import { ethers } from 'hardhat';
 import { given, then, when } from '@utils/bdd';
 import { SignerWithAddress } from '@nomiclabs/hardhat-ethers/signers';
 import { JsonRpcSigner } from '@ethersproject/providers';
-import { IERC20, SwapperRegistry, SwapProxy } from '@typechained';
-import { BigNumber, utils } from 'ethers';
+import { IERC20, IERC20Permit, SwapperRegistry, SwapProxy } from '@typechained';
+import { BigNumber, constants, utils } from 'ethers';
 import { snapshot } from '@utils/evm';
+import { fromRpcSig } from 'ethereumjs-util';
 import { expectBalanceToBeEmpty, expectBalanceToBeGreatherThan } from './assertions';
 import { deployContractsAndReturnSigners, getQuoteAndAllowlistSwapper } from './utils';
 import { oneInchAdapter, paraswapAdapter, Quote, QuoteInput, zrxAdapter } from './dex-adapters';
 
-const AMOUNT_EXACT_IN = utils.parseEther('1');
```

Comment:
Before this change, WETH was the sellToken and USDC was the buyToken. We changed it so that we could test the permit extensions